scout/server/blueprints/panels/templates/panels/panel.html

```diff
@@ -105,7 +105,8 @@
       {% endif %}
       <li class="list-group-item">
         <div class="d-flex justify-content-between">
-          <a href="{{ url_for('panels.panel_export', panel_id=panel._id) }}" class="btn btn-secondary btn-sm" download>Export to PDF</a>
+          <a href="{{ url_for('panels.panel_export_pdf', panel_id=panel._id) }}" class="btn btn-secondary btn-sm" download>Export to PDF</a>
+          <a href="{{ url_for('panels.panel_export_txt', panel_id=panel._id) }}" class="btn btn-secondary btn-sm" download>Export to TXT</a>
```

Comment:
A little tooltip bragging that exported TXT panels are in a format that can be imported into other institutes or Scout instances perhaps?

Reply:
Yes, that would be nice!

Reply:
Done: 
<img width="780" alt="image" src="https://user-images.githubusercontent.com/28093618/225595716-ca31e781-c80a-40ba-955c-7c245fb22655.png">